File: CourseRegistrationSystem/src/Grade.java
public class Grade {
    private Course course;
    private String gradeValue;

    public Grade(Course course, String gradeValue) {
        this.course = course;
        this.gradeValue = gradeValue;
    }

    public Course getCourse() {
        return course;
    }

    public String getGradeValue() {
        return gradeValue;
    }

    @Override
    public String toString() {
        String gradeDatas = String.format("%-10s %-30s %-10s\n",
                getCourse().getCourseId(),
                getCourse().getCourseName(),
                getGradeValue());
        return gradeDatas;
    }
}

File: iteration3/CourseSystem/Grade.py
class Grade:
    def __init__(self, course=None, grade_value=""):
        self.course = course
        self.grade_value = grade_value

    def get_course(self):
        return self.course

    def get_grade_value(self):
        return self.grade_value

    def __str__(self):
        grade_data = f"{self.course.course_id:<10} {self.course.course_name:<35} {self.grade_value:<10}\n"
        return grade_data
